content/en/serverless/azure_container_apps/_index.md

```diff
@@ -84,7 +84,7 @@ See [Tracing Java Applications][1] for detailed instructions. [Sample code for a
 {{< /programming-lang >}}
 {{< programming-lang lang="dotnet" >}}
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
```

Comment:
did you test that? I assume this is going to fetch the `latest` but just want to double check :)

Reply:
yep
```sh
 docker pull datadog/serverless-init      
Using default tag: latest
latest: Pulling from datadog/serverless-init
b5064dc4d3d2: Already exists 
Digest: sha256:cb24d96298587f7f1fc449107b13181572e96e6439762785afaccff192d1e31a
Status: Downloaded newer image for datadog/serverless-init:latest
docker.io/datadog/serverless-init:latest
```